cmd/web.go

```diff
@@ -48,6 +48,26 @@ and it takes care of all the other things for you`,
 	},
 }
 
+func runHTTPRedirector() {
+	source := setting.HTTPAddr + ":80"
```

Comment:
Also don't assume port 80 😉

Reply:
The port could be another config on app.ini